tendrl/gluster_integration/message/gluster_native_message_handler.py

```diff
@@ -55,12 +55,16 @@ def _setup_gluster_native_message_reciever(self):
         cmd = cmd_utils.Command('gluster-eventsapi webhook-add %s' % url)
         out, err, rc = cmd.run()
         if rc != 0:
+            severity = "info" if err == "Webhook already exists" else "error"
```

Comment:
Shouldnt this be like below?

`severity = "info" if "Webhook already exists" in err else "error`

Reply:
ack